Almost final version ;-)

w-jan opened the pull request.

w-jan committed:
git-ignore add local data folder with large binary data

w-jan committed:
tube_selection.Rmd start up

w-jan committed:
tube_selection.rmd :  load habmap

w-jan committed:
tube_selection.Rmd progressing setting up survey: selection GTRS-cells in relation to the relative surface of a (one of five) groundwater-group

w-jan committed:
tube_selection.Rmd till tube_selection

florisvdh committed:
File renamed (tidyverse style guide)

florisvdh committed:
Don't load obsolete n2khabutils package

florisvdh committed:
Use n2khab functions to read habitatmap & types

florisvdh committed:
Correct term milieudruk-schema's

florisvdh committed:
Fix markdown titles syntax

florisvdh committed:
Comment out the line that sets CRS to epsg:31370

w-jan committed:
tube_selection: check update n2khab-package - branch set_crs. CRS is now correct (31370)

w-jan committed:
tube_selection:
some text layout changes in favour of RMarkdown syntax

w-jan committed:
Merge branch 'tube_selection' into tube_sel_fv

w-jan committed:
Merge pull request #1 from w-jan/tube_sel_fv

Some smaller updates

w-jan committed:
tube_selection: get more insight in the distribution of tubes with xg3-values concerning the selected raster-cells

w-jan committed:
tube_selection.Rmd
extraction of tubes for which it's worthwile to explore their timeseries

w-jan committed:
tube_selection.Rmd: further progress

w-jan committed:
saving tubes_hab as Git2rdata

w-jan committed:
tube_selection.Rmd: working on selection and cluster criteria based on number year-timeseries and on the most recent timeserie

w-jan committed:
data storage: tubes_hab and tubes_xg3

w-jan committed:
tube_selection: resulting data

w-jan committed:
tube_selection: progress on
selecting tubes in group 4 (excess of tubes)
selecting habitatpatches in group 1 (no pb's)

w-jan committed:
add data tubes_pot_group1

w-jan committed:
tube_selection.Rmd: first complete run

w-jan committed:
Upgrade some Rmd-files (for bookdown)
00_setup.Rmd
01_Inleiding.Rmd
02_Methodiek.Rmd
index.Rmd
lit_droogtemeetnet.bib

w-jan committed:
work in progress
03_Uitvoering.Rmd
droogtemeetnet_code.R

w-jan committed:
update data-files

w-jan committed:
updates:
index.rmd
02_Methodiek.Rmd
03_Uitvoering.Rmd
droogtemeetnet_code.r

w-jan committed:
update
02_methodiek.Rmd
03_uitvoering.Rmd
index.Rmd
droogtemeetnet_code.r

w-jan committed:
update/adding data concerning the tube selection

florisvdh committed:
Rmd 01: use read_habitatmap_terr()

florisvdh committed:
Rmd 02: remarks on extra quality criteria

florisvdh committed:
Rmd 02: remark on catchment area level

florisvdh committed:
Rmd 02/03: more clearly refer to GRTSmaster_habitats etc.

florisvdh committed:
Take advantage of units, e.g. use 'ha' in tables

florisvdh committed:
Rmd 03: use connect_watina() again

florisvdh committed:
Rmd 03: don't disconnect and reconnect watina

florisvdh committed:
Rmd 01: refer to synergy with MNE

florisvdh committed:
Use INBO styled citations & bibliography

florisvdh committed:
Various small updates, fixes, enhancements

florisvdh committed:
Update gitignore

florisvdh committed:
Add _bookdown.yml to set the output filename

w-jan committed:
Merge pull request #2 from inbo/tube_sel_fv

Various small additions & updates

w-jan committed:
update 02_Methodiek.Rmd

w-jan committed:
Merge branch 'tube_selection' of https://github.com/inbo/n2khab-droogtemeetnet into tube_selection

w-jan committed:
Rmd-updates based on Floris' suggestions
concerning use of 'units' and of a more distinctive colorpalette.
I had to disable the use read_habitatmap_terr-function because of an error.

w-jan committed:
adding data-results

w-jan committed:
update xg3-data

w-jan committed:
adding Menyanthes report

w-jan committed:
update Menyanthes report

w-jan committed:
update Menyanthes report

w-jan committed:
update Rmd-files concerning Menyanthes
02_Methodiek
03_Uitvoering

w-jan committed:
update Menyanthes report

w-jan committed:
update resultaten Menyanthes

w-jan committed:
update
02_Methodiek.Rmd
03_Uitvoering.Rmd
_bookdown.yml

w-jan committed:
data:
update sourcefile tblTubes_Menyanthes_report
update some resultsfiles: tubes_catxx

w-jan committed:
solved some selection_issues.
Hopefully the
rastercells selection and the pools of tubes, for further selection with GTRS-strategy, are ready.

w-jan committed:
update .gitignore cfr master

Merge branch 'master' into tube_selection

# Conflicts:
#	.gitignore

w-jan committed:
update 03_Uitvoering.Rmd

w-jan committed:
Adding some more explanation about the categorization of the rastercells in 02_Methodiek.Rmd

w-jan committed:
adding some more clarification about the categorization in 02_Methodiek.Rmd

florisvdh committed:
02_Methodiek.Rmd: fix some typos or make things a bit more clear

w-jan committed:
update all Rmd-files:
00_setup: the code to import the datasources (eg from databases) are collected in two chunks. The datasources are copied to local drives where it can be versionised or to a google drive.

01_inleiding:
Code to read the datasource via local pathways.

02_methodiek: some minor changes

03_uitvoering:
Assess all tubes belonging to cat 1B (in stead of only the tubes that have a lg3-value)
Description for making the grts-selection
Code to do that
Indicate the reserve points

w-jan committed:
update data

w-jan committed:
update 2 Rmd-files
00_setup, 03_Uitvoering: separate reading in two phases (continuation of the preceded commit)

w-jan committed:
some minor updates for 3 Rmd-files:
00_setup: add googledrive-library
01_inleiding
03_uitvoering: add the possibility to exclude tubes

w-jan committed:
adding data

w-jan committed:
99_sessioninfo

w-jan committed:
03_Uitvoering.Rmd: some adjustments on the primary habitat-tube overlay, because some tubes were counted more than once.

w-jan committed:
update data: especially adding two tube characteristics (filterdepth and surfacelevel)

w-jan committed:
update 00_setup
using the latest version of read_habitatmap_terr

w-jan committed:
saving some workfiles

w-jan committed:
Update 3 Rmd.files mainly about reading and writing data
00_setup.Rmd
01_Inleiding.Rmd
03_Uitvoering.Rmd

w-jan committed:
update data

w-jan committed:
03_uitvoering.rmd : a few corrections

w-jan committed:
02_Methodiek: more clarification about classification
03_Uitvoering: minor change

w-jan committed:
Merge pull request #4 from inbo/tube_sel_fv

02_Methodiek.Rmd: fix some typos or make things a bit more clear

w-jan committed:
renaming droogtemeetnet_code2 -> droogtemeetnet_script to emphasize that it contains only the code that's being used in de bookdown-version

w-jan committed:
some Rmd-updates and gitignore-update
00_setup: to handle making time-consuming figures
02_methodiek: some clarification about categories
03_Uitvoering: some small updates

w-jan committed:
update 02_Methodiek
solving conflicts after merging tube_selection

Merge branch 'tube_selection' into grts_selection

# Conflicts:
#	02_Methodiek.Rmd

w-jan committed:
small update Rmd
00_setup: refresh
01_inleiding: fix typo
03_uitvoering: handling time-consuming figure

w-jan committed:
03_uitvoering: fix typo

florisvdh committed:
Add header.html in repo (& allow in gitignore)

florisvdh commented on 00_setup.Rmd:
```diff
@@ -61,6 +83,316 @@ minaantal_tub_group <- as.integer(tot_n_tub/aantal_strat)
+  
+  #inlezen grts-raster level 1 (hoogste resolutie = kleinste gridcelgrootte)
+  grts_level1 <- read_GRTSmh(brick = TRUE) %>% 
+    raster::subset(1)
```
zonder al de hele context te kennen (mijn oog valt er even op); als je hier doelt op het fijnste raster (met elke 32 m cel een ander adres), dan heb je level 0 nodig, niet level 1 (64 m resolutie). Maar ik vermoed dat het een bewuste keuze is

florisvdh replied:
level 0 krijg je ook met plain `read_GRTSmh()`

florisvdh replied:
Mijn frank valt nu wat je doet. Er is geen probleem. Wel wat verwarring. In het resultaat van `read_GRTSmh(brick = TRUE)` zijn de brick layers gelabeld van `level0` tot `level9`. `subset(1)` is dus level0 (32m); je benoemt het als level1. Analoog voor level8 (resol 8192 m) = `subset(9)`

Je kunt dus ook gewoon als volgt werken: `... <- read_GRTSmh(brick = TRUE)$level0`

florisvdh replied:
de levelnummers hebben te maken met de exponent van factor 2 in `32 * 2^(0:9)`

w-jan replied:
Ik heb de naamgeving logischer gemaakt level1 -> level0; level9 -> level8

florisvdh commented on 00_setup.Rmd:
```diff
@@ -34,6 +34,28 @@ library(tmap)
+library(googledrive)
+library(RSQLite) 
+
+#keuzen voor het aanmaken van een document
+#welke data verversen
+# 2: alles; 1: enkel de huidige google-drive-bestanden downloaden ; 0: enkel met lokale data
+refresh_data <- 0
+if (refresh_data == 2) {
+  datapath <- "G:/Mijn Drive/PRJ_Meetnet_Droogte/2_Uitvoering/data"
+  dir.create(file.path(datapath, "GIS/VoorR/"), recursive = TRUE)
+}
+
+#figuren opnieuw aanmaken of inlezen
+# 2: figuren terug aanmaken (en wegschrijven); 1: de huidige figuren van google-drive downloaden, 0: alleen lokale figuren inlezen
+refresh_figures <- 0
+if (refresh_figures == 2) {
+  figpath <- "G:/Mijn Drive/PRJ_Meetnet_Droogte/2_Uitvoering/figuren"
+}
```
knap hoe je dit uitgedokterd hebt.

florisvdh committed:
00_setup.Rmd: create local gitignored directories first

florisvdh reviewed (commented):
Just a few intermediate comments.

I also propose some small updates in PR #6 just for bookdown, so you can merge them in your branch here.

I could produce the whole html :smile: !!

w-jan committed:
Merge pull request #6 from inbo/grts_sel_fv

Small fixes for bookdown reproducibility

w-jan committed:
update 3 Rmd-files:
- especially adding calculation selection habitatpolygons for a future tube (cat. 1A)
- renaming grid-levels

00_setup, 01_inleiding: cleaning chunk-options
03_Uitvoering:
- adding synthese tube-selection
- making a polygon-selection
- changing chunck-options (refer more to general options)
- renaming grid-object
-

w-jan committed:
data: tubes_cat2_bis (include some more variables, tube-characteristics)

florisvdh committed:
git-ignore droogtemeetnet_ontwerp.md

florisvdh commented on 00_setup.Rmd:
```diff
@@ -61,6 +84,326 @@ minaantal_tub_group <- as.integer(tot_n_tub/aantal_strat)
+  
+  habmap_types <- habmap_terr$habitatmap_terr_types
+  
+  habmap_types <- habmap_types %>%
+    mutate( polygon_id = as.factor(.data$polygon_id),
+            certain = .data$certain == 1,
+            type = factor(.data$type,
+                          levels = levels(types$type)
+            )
+    )
```
overbodig stuk? Dit lijkt me code uit `read_habitatmap_terr()`, die hier minstens in het geval van de kolom `certain` een ongewenst gedrag zal geven (want `certain` is al een TRUE/FALSE)

florisvdh commented on 00_setup.Rmd:
```diff
@@ -61,6 +84,326 @@ minaantal_tub_group <- as.integer(tot_n_tub/aantal_strat)
+  
+  habmap_polygons_gw <- 
+    habmap_polygons %>% 
+    inner_join(habmap_types_gw %>% 
+                 dplyr::select(-code_orig), 
+               by = "polygon_id")
```
ik zou gaan voor `semi_join()`, niet? Via `semi_join` zul je geen polygonen verdubbelen, met `inner_join` wel.

florisvdh replied:
het feit dat je verder een GPKG met twee tabellen maakt, lijkt me dit te bevestigen. Via een inner_join zou je alles bijeen steken en is een tweede tabel overbodig. Maar met duplicatie van polygonen.

florisvdh commented on 00_setup.Rmd:
```diff
@@ -61,6 +84,326 @@ minaantal_tub_group <- as.integer(tot_n_tub/aantal_strat)
+  raster_meetnet_poly <- raster_meetnet_poly %>% 
+    rename(rasterid =  value)
+  
+  #wegschrijven van dit grid
+  st_write(raster_meetnet_poly,
+           file.path(datapath, 
+                     "GIS/VoorR/raster_meetnet_poly.gpkg"), 
+           layer = "raster_meetnet_poly", 
+           driver = "GPKG",
+           delete_dsn = TRUE)
```
Op zich niets mis mee (kun je gerust laten zijn); is dit om op eventuele 'externe' evoluties te anticiperen en het nog meer 'self-contained' te maken? Anders zie ik niet meteen waarom je dit lokaal wegschrijft. (deze opmerking geldt even goed voor enkele andere datalagen die je wegschrijft) De brondata van de GRTS-inleesfuncties staan [hier](https://zenodo.org/communities/n2khab-data-processed) al op Zenodo, dus die zullen altijd beschikbaar blijven. `n2khab`-versies zullen dat in principe ook altijd zijn (via github dan) - het plan is om toekomstige releases ook op Zenodo te gaan stockeren (kan automatisch met github gekoppeld worden).
Meerwaarde van lokaal wegschrijven lijkt me vooral te bestaan wanneer de processing-stappen veel CPU-tijd kosten, bv. wellicht zo bij `habmap_gw_raster_overlay`, of wanneer we geen controle hebben over de evolutie van de bron (bv. Watina).

florisvdh commented on 00_setup.Rmd:
```diff
@@ -61,6 +84,326 @@ minaantal_tub_group <- as.integer(tot_n_tub/aantal_strat)
+  tubes_hab <- tubes_hab %>% 
+    arrange(loc_code, polygon_id, rasterid, type)
+  
+  output_vc <- write_vc(tubes_hab, file.path(".","data","tubes_hab"), 
```
> file.path(".","data","tubes_hab")

dit is super-veilig gecodeerd (doe je op veel plaatsen). Louter ter info (geen aanpassing nodig) - volgens mij werkt gewoon het gebruik van de string `"data/tubes_hab"` even goed. Alleszins niet nodig om `"."` of "`./"` te prefixen voor directory- of filepaths vanaf working directory.

florisvdh commented on 00_setup.Rmd:
```diff
@@ -61,6 +84,326 @@ minaantal_tub_group <- as.integer(tot_n_tub/aantal_strat)
+  tubes_xg3 <- tubes_xg3 %>% 
+    arrange(loc_code, hydroyear)
+  
+  output_vc <- write_vc(tubes_xg3, file.path(".","data","tubes_xg3"), 
+                        sorting = c("loc_code", "hydroyear"),
+                        strict =  FALSE, root = ".")
```
Itt tot bv. GRTS layers is wegschrijven hier inderdaad zeer relevant, aangezien we niet zo eenvoudig aan een geversioneerde toestand van Watina kunnen.

florisvdh commented on 00_setup.Rmd:
```diff
@@ -61,6 +84,326 @@ minaantal_tub_group <- as.integer(tot_n_tub/aantal_strat)
+  
+  DBI::dbDisconnect(watina)
+  
+  #volgende watina-functie werkt ook zonder databankconnectie, maar omdat deze op sommige pc's niet stabiel werkt, worden de data hier alvast lokaal weggeschreven.
+    
+filter_xg3_lok <- function(data,
+                           xg3_type) {
```
deze en verdere functies: ik heb het niet nader uitgeplozen / vergeleken met watina; is de essentie hier dat je ze laat werken op lokale tibbles ipv een databank-connectie of zoiets? Is dan inderdaad relevant. Ook andere updates? Als je meent dat deze functionaliteit vaker nodig is, dan is het misschien een piste om de watina-functies in die zin verder te updaten met een optie om lokaal te werken; ter overweging! (je kan altijd updates voorstellen met een PR, bv. voor één functie om te beginnen).
Om Rmd-files niet te zwaar te maken, kan het soms fijn zijn om dgl. functiedefinities te groeperen in een R-script (dat je ook van elders kunt sourcen), en in de Rmd dit script te sourcen met `source()`.

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -59,45 +59,63 @@ Het kan dan zijn dat het bekomen aantal lichtjes hoger uitvalt.
+_**Of het tegenovergestelde is ook een alternatief: dat voor de ruimtelijke balancering we geen voorkeur geven aan meetpunten met een peilbuis.**_
+_**We vinden dat het huidige voorstel een goede balans vertoont tussen beide.**_
+
+Het gaat hier vooral over een tijdsinvestering: dergelijke meetpunten zullen pas na enkele jaren effectief meedraaien in het meetnet. 
```
deze zin slaat specifiek op de nieuw te installeren peilbuizen, maar die connectie is niet (meer) duidelijk nu

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -108,36 +126,48 @@ We beschouwen een meetpunt als geschikt voor het berekenen van een droogte-indic
 Er zijn momenteel maar twee peilbuizen beschikbaar, één met een goede tijdreeks en één met een slechte. 
-De rastercel zal dan behoren tot categorie 1A: er dient nog een meetlocatie bij gezocht te worden, tot categorie 1B: de peilbuis met een slechte tijdreeks komt in de wachtrij en tot categorie 2: de peilbuis met een goede tijdreeks wordt sowieso geselecteerd voor onmiddellijke ingebruikname.
+De drie locaties vragen elk een andere benadering. 
+Voor het meetpunt zonder peilbuis, moet op zoek gegaan worden naar de meest presentatieve keuze. 
```
```suggestion
Voor het meetpunt zonder peilbuis, moet op zoek gegaan worden naar de meest representatieve keuze. 
```

florisvdh replied:
hier en ook elders: 'meetpunt zonder peilbuis' lijkt te suggereren dat je de locatie ervan al kent ('meetpunt'). Maar is vermoedelijk niet zo. Misschien duidelijker om te spreken van een 'nog ontbrekend meetpunt', ...?

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -108,36 +126,48 @@ We beschouwen een meetpunt als geschikt voor het berekenen van een droogte-indic
 
 
-[^1]: lg3 =  het gemiddelde van de drie laagste waterpeilen binnen een hydrologisch jaar (1 april - 31 maart), waarbij er tussen deze drie metingen minstens 14 dagen verschil ligt.
+[^2]: lg3 =  het gemiddelde van de drie laagste waterpeilen binnen een hydrologisch jaar (1 april - 31 maart), waarbij er tussen deze drie metingen minstens 14 dagen verschil ligt.
```
> [^2]:

voor de zekerheid dat je het weet: het mogen ook labels zijn ipv nummers. Voetnootnummering gebeurt automatisch

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -108,36 +126,48 @@ We beschouwen een meetpunt als geschikt voor het berekenen van een droogte-indic
+De drie locaties vragen elk een andere benadering. 
+Voor het meetpunt zonder peilbuis, moet op zoek gegaan worden naar de meest presentatieve keuze. 
+De twee andere meetlocaties kunnen opgenomen worden, maar de ene locatie komt in de wachtrij te zitten, terwijl met de andere effectief al van start kan gegaan worden. 
+Om dit technisch te kunnen ondervangen, zal de rastercel tot drie categorieën gerekend worden (1A, 1Ba en 2).
```
Mooi verwoord :+1:

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -108,36 +126,48 @@ We beschouwen een meetpunt als geschikt voor het berekenen van een droogte-indic
+- **Cat. 1Ba**: het aantal ongeschikte meetpunten is onvoldoende tot juist voldoende om alle gewenste locaties te kunnen invullen.
+- **Cat. 1Bb**: het aantal ongeschikte meetpunten is groter dan het aantal gewenste locaties: er zal hiertussen nog een keuze moeten gemaakt worden.
+- **Cat. 2**: de vraag is groter of gelijk aan het aanbod van peilbuizen van goede kwaliteit: er zijn geschikte meetpunten, juist voldoende of te weinig: elk meetpunt wordt geselecteerd en kan onmiddellijk ingezet worden.
+- **Cat. 3**: de vraag is kleiner dan het aanbod van peilbuizen van goede kwaliteit: er is een overschot aan geschikte meetpunten
```
op zich duidelijk verwoord :+1: 
Wel blijft het voorlopig nog een mysterie waarom een rastercel tot meerdere categorieën zou kunnen behoren

florisvdh replied:
het begint me een beetje te dagen, na heen en weer kijken naar je twee voorbeelden. In feite ga je groepen van meetpunten binnen een rastercel onderscheiden, iets in de richting van:

- er zijn x goede
- er zijn y 'slechte'
- er zijn z - x goede tekort en z-x kan > of < zijn dan y, en z-x kan ook < 0

florisvdh replied:
Zie nog [deze opmerking](https://github.com/inbo/n2khab-droogtemeetnet/pull/5#discussion_r344430340).

florisvdh replied:
Op basis van latere lezing (zie latere review), voeg ik hier een tentatieve definitie in op niveau van de rastercel, met behulp van x, y, z (z = totaal aantal nodig), en geënt op je bovenstaande beschrijving

- 1A: x < z en y = 0
- 1B: x < z en y > 0
  - 1Ba: x + y <= z
  - 1Bb: x + y > z
- 2: x <= z en y >= 0. Waarom de overlap met de voorgaande?
  - deze definitie is in overeenstemming met (regel 792 van Rmd 03): "_Als de kwaliteit van de tijdreeks van een aantal peilbuizen voldoende is om onmiddellijk in een meetnet te kunnen worden opgenomen, wordt de cel ook tot categorie 2 gerekend._" Echter: waarom deze complexiteit van iets tot 2 categorieën rekenen? De term 'categorieën' impliceert m.i. reeds 'wederzijds uitsluitend', dus geen overlap.
  - anderzijds, onder de kop 'het opleggen van bijkomende kwaliteitscriteria' (Rmd 03) staat dan weer: "_Er kan dan een groep cellen onderscheiden worden waarbij het gewenste aantal meetpunten juist overeenstemt met het aantal beschikbare peilbuizen die gelijkwaardig van kwaliteit zijn. Deze rastercellen behoren tot categorie 2._" Dat wijst op een definitie die veel logischer past in het rijtje: **x = z**. Dus: consequent maken. Als je 't mij vraagt, deze categorie beperkten tot x = z; overlap lijkt me onnodig complex.
- 3: x > z

_Het lijkt me een goed idee om, louter voor de uitleg, de peilbuizen van een GTgroep binnen een cel in klassen in te delen (goede, slechte, evt. meer, alleen als dat nodig is). Vervolgens de celcategorieën definiëren op basis van de onderlinge relatie tussen de frequentie van die klassen (cf. x, y hierboven) en op basis van de relatie met het gewenste aantal meetpunten (z hierboven) eenvoudige formules als hierboven._ Tekstueel is het m.i. moeilijker te volgen dan met een formulevorm. Zit daar iets in?

Voorts, als volgens de definities x = z en x > z geen cellen tot cat 2 en 3 behoren, dan kan dit bij de uitwerking zo gesteld worden en hoef je alleen verder met 1A en 1B (wat je alleszins doet), zonder die laatste twee verder in relatie te brengen met 2 en 3. (In het andere geval: dan was me niet duidelijk dat dit zo is).

Zie ook een aantal opmerkingen bij de review in PR #3 .

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -108,36 +126,48 @@ We beschouwen een meetpunt als geschikt voor het berekenen van een droogte-indic
 
 Voorbeeld 2: in een rastercel zijn drie meetpunten gewenst. 
 Er zijn vijf peibuizen beschikbaar, twee met kwaliteitsvolle tijdreeksen, de drie andere met tijdreeksen van onvoldoende kwaliteit. 
 Deze rastercel zal behoren tot categorie 2 voor de twee peilbuizen van goede kwaliteit en tot categorie 1B: voor het derde meetpunt moet uit de drie peilbuizen met onvoldoende kwaliteit er één nog gekozen worden.
```
Misschien nog iets als '(zie verder)' toevoegen aangezien de cat-definitie nog komt.

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -242,10 +274,17 @@ $$trend - 2* sd \leq 1(cm/jaar)$$
-Sommige tijdreeksen werden bijna perfect gefit, maar omdat het aantal beschikbare metingen maar op één meetjaar sloeg, was het toch onbetrouwbaar om het model te extrapoleren over een relatief lange tijdsperiode. 
-Anderzijds werd een model met een verklarende variantie < 66% toch soms nog weerhouden, als het model goed de lage grondwaterstanden kon modelleren.
-Zo kon ook een model met een niet significante trend, maar met een heel geringe fluctuaties toch afgekeurd worden en vice versa.
+Bij de modelselectie kwam toch ook nog expertoordeel bij kijken. 
```
```suggestion
Bij de modelselectie kwam toch ook nog expertoordeel kijken. 
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -242,10 +274,17 @@ $$trend - 2* sd \leq 1(cm/jaar)$$
+
+Er werden geen kwaliteitseisen gesteld aan een meetreeks die deze a priori zou uitsluiten van een tijdreeksanalyse. 
+De gestelde eisen hebben alleen een weerslag op de categorie-indeling. 
+In principe kan elke peilbuis in categorie 1B terecht komen, zelfs is er maar één meting van bekend.
```
de categorieën lijken volgens deze zin meetpunt-gebonden (rekening houdend met andere meetpunten in rastercel), niet op niveau rastercel zoals eerder gedefinieerd. Dat is niet in overeenstemming met de definitie, noch hoe het uitgevoerd wordt. Dus consistent maken met elders.

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -242,10 +274,17 @@ $$trend - 2* sd \leq 1(cm/jaar)$$
+De gestelde eisen hebben alleen een weerslag op de categorie-indeling. 
+In principe kan elke peilbuis in categorie 1B terecht komen, zelfs is er maar één meting van bekend.
+En zo geschiedde ook.
+Deze meetreeksen werden manueel afgekeurd. Sommige van deze tijdreeksen hadden bijna een perfecte fit, maar het was toch onbetrouwbaar om hiermee waarden te simuleren over een relatief lange tijdsperiode. 
```
> Er werden geen kwaliteitseisen gesteld aan een meetreeks die

> werden manueel afgekeurd.

Op een of andere manier heeft de 'manuele afkeuring' een bepaalde regel gevolgd. Wellicht kan die regel code-matig geïmplementeerd worden zodat een pre- of postselectie voor/van tijdsreeksanalysen kan gebeuren?

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -242,10 +274,17 @@ $$trend - 2* sd \leq 1(cm/jaar)$$
+
+Anderzijds werd een model met een verklarende variantie beneden 66% soms nog weerhouden, indien bleek dat het model de lage grondwaterstanden goed kon modelleren.
+
+Een tijdreeks met een trend < 1cm/jaar, maar waarbij de peilen maar weinig schommelen (kleine amplitude) kon toch afgekeurd worden en vice versa.
```
```suggestion
Een tijdreeks met een (absolute) trend < 1cm/jaar, maar waarbij de peilen maar weinig schommelen (kleine amplitude) kon toch afgekeurd worden en vice versa.
```

florisvdh reviewed (commented):
More comments on Rmd's 00 and 02. Fine work @w-jan .

More is to come later (03).

florisvdh committed:
03_Uitvoering: replace rasternr by rastercelnr

florisvdh commented on 03_Uitvoering.Rmd:
```diff
@@ -743,12 +788,12 @@ Figuur \@ref(fig:cat1A-plot) toont ze op een kaart.
-Indien een rastercel wel enkele peilbuizen heeft, weliswaar onvoldoende, dan zal deze rastercel tot meerdere categorieën behoren, afhankelijk van de kwaliteit van de meetreeksen. 
-Als de kwaliteit van de tijdreeks van een aantal peilbuizen voldoende is om al onmiddellijk in een meetnet te kunnen worden opgenomen, wordt de cel ook tot categorie 2 gerekend. 
+Indien een rastercel wel enkele peilbuizen heeft, weliswaar onvoldoende, dan zal deze rastercel tot meerdere categorieën behoren, afhankelijk van de kwaliteit van de meetreeksen (zie \@ref(indeling-cat)). 
+Als de kwaliteit van de tijdreeks van een aantal peilbuizen voldoende is om onmiddellijk in een meetnet te kunnen worden opgenomen, wordt de cel ook tot categorie 2 gerekend. 
```
Deze alinea staat onder cat. 1A, terwijl het mij  over situatie 1B lijkt te gaan (y > 0).

florisvdh committed:
03_Uitvoering: clarification (to be checked by @w-jan)

florisvdh committed:
03_Uitvoering: update title 1A (to be checked by @w-jan)

florisvdh committed:
03_Uitvoering: bugfix explanation of cat 1B (to be checked by @w-jan)

florisvdh committed:
03_Uitvoering: clarification (to be checked by @w-jan) *

Term 'clusters' was used before to denote the combination,
so don't use the term cluster here to denote individual
criteria grouping.

florisvdh committed:
03_Uitvoering: improve description of level 8

florisvdh committed:
03_Uitvoering: minor fixes, incl markdown syntax

florisvdh committed:
02_Methodiek: attempt to fix markdown syntax, to be checked in compilation

florisvdh committed:
Define appendices part and put 'used environment' in it

florisvdh committed:
Rename sessioninfo appendix for ordering reasons

florisvdh committed:
Add an unnumbered references chapter at the end

florisvdh committed:
Fix incorrect bibtex author syntax

florisvdh reviewed (commented):
This review is about Rmd 03 and is spread over two actual reviews. I hope it's useful to you!

- It adds just 1 comment here, however this is because most of the work in 03 sits in the `tube_selection` branch, i.e. PR #3 . That is because review comments can only be made on the 'diff' that is defined by the two branches to be merged by this PR. It would actually have been better to either keep everything together in the same branch, or in any way have just one PR, namely of the latest branch directly to master (this would also involve the commits of `tube_selection`). No worries however.
- Related hindsight comments on Rmd 02 were added to my previous review above, or those comments were further edited.
- I've read the largest part, i.e. up to "_De rastercellen hercategoriseren na de tijdreeksanalysen_". As I understand it, you apply a reasonable approach to select existing locations for the drought monitoring, within the limitations of what is available and what is not, and with the limitation of a low number of location selection to represent a GTgroup. Those limitations can be potentially severe for 'extrapolation' to the target population; we simply cannot say.
- My concerns, as you'll see, are mainly about the categories. Perhaps they even receive too much attention: what you actually do seems sensible (note, I did not review most of this R-code, but read the html). Anyhow: to me there seem inconsistencies in how they are defined, redefined are named throughout the document, which causes a lot confusion. I propose simplifications here and there; the main proposal is addressed in [this comment](https://github.com/inbo/n2khab-droogtemeetnet/pull/5#discussion_r344455925) in the previous review on Rmd 02.
- Another comment is on report structure. I believe it could be good to have the elaborate part in the appendices, and focus more on end results in the report itself, in order to keep it short (and easy) enough. But you decide; we can see later on.

You find an additional PR with my commits in PR #7, which you can merge in your branch if it's all fine.

w-jan committed:
fix typo

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

w-jan committed:
a major update

especially concerning 02_Methodiek and 03_uitvoering.
the main goal was to make the methodology less complex.
Cat. 1B was dropped. The distinction between good and 'bad' timeseries can be made at the of the workflow.

w-jan committed:
update data

1 source file : menyanthes-report
a lot resulting data

w-jan committed:
small update

Merge branch 'grts_selection' of https://github.com/inbo/n2khab-droogtemeetnet into grts_selection

# Conflicts:
#	_02_Methodiek.Rmd

w-jan committed:
update 02_Methodiek_vereenvoudigd

adding workflow-schemes

w-jan committed:
03_Uitvoering_vereenvoudigd

adding adjusment GRTS-selection: when the number of good time-series is greater than the desired number, only the good time-series will be used in the selection

w-jan committed:
renaming 2 Rmd-files: O2_Methodiek and 03_Uitvoering

w-jan committed:
Merge branch 'grts_selection' into grts_sel_fv

w-jan committed:
Merge pull request #7 from inbo/grts_sel_fv

Some additions, fixes etc

w-jan committed:
update 02_Methodiek.Rmd improve the base-criteria

w-jan committed:
update 03_Uitvoering.Rmd according to new definition of cat. 2 and cat. 3

w-jan committed:
update 02_Methodiek.Rmd (small) and 03_Uitvoering.Rmd (tube selection cat 3)

w-jan committed:
update data

w-jan committed:
update figures

w-jan committed:
small update 03_uitvoering.Rmd

figure with potential measure locations

w-jan committed:
Update figure workflow (part2)

florisvdh commented on 01_Inleiding.Rmd:
```diff
@@ -65,7 +67,8 @@ kable(gw_types_groupen,
+_ **_De vraag is of we deze groep meenemen._**  
 De GT-groepen lenen zich heel goed om een stratificatie uit te voeren op basis van de verwachte responssnelheid van een locatie op droogte. 
 De permanent gevoede locaties (GT-groep 1) zullen naar verwachting trager reageren dan de tijdelijk of zwak gevoede locaties. 
 Door een stratificatie over de GT-groepen toe te passen kan een _zekere_ **_(nog af te spreken)_** balans worden verzekerd. 
```
hier kun je ook de overweging meegeven of we per groep uitspraken willen doen (lijkt me zinvol), dan wel (stratificatie s.s.) alleen overall

w-jan replied:
OK, dit als discussiepunt toegevoegd.

florisvdh commented on 01_Inleiding.Rmd:
```diff
@@ -65,7 +67,8 @@ kable(gw_types_groupen,
 
-_GT-groep 5 bevat uitsluitend grondwaterafhankelijke HT die zodanig breed gedefinieerd zijn dat ze op sommige locaties grondwatergevoed zijn, maar soms tot meestalniet._ **_De vraag is of we deze groep al dan niet meenemen._**  
+_GT-groep 5 bundelt grondwaterafhankelijke HT die zodanig breed gedefinieerd zijn dat ze op sommige locaties grondwatergevoed zijn, maar soms tot meestal niet.
+_ **_De vraag is of we deze groep meenemen._**  
```
Best meteen hierna een zin plaatsen die aangeeft wat het voordeel (+ nadeel?) is van groep 5. Voordeel is alvast snelle reactie op droogte. Bij nieuwe locaties selecteren (nieuwe peilbuizen) is de voorwaarde alvast dat een locatie grondwatergevoed moet zijn (shallow groundwater), dus in groep 5 kan dat een (beperkt?) nadeel geven.

w-jan replied:
OK, heb dit als discussiepunt toegevoegd

florisvdh reviewed (commented):
Few comments on Rmd 01.
Starting with Rmd 02 now.

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+Idealiter liggen de meetlocaties ruimtelijk gebalanceerd, omdat dit de beste garantie biedt dat de gebeurtenissen die zich op deze locaties voordoen, representatief zijn voor het hele gebied.   
+Men kan een ruimtelijk gebalanceerde set opbouwen door de verspreiding van de GT-groepen over een fijn GRTS-raster[^1] te leggen. 
+
+[^1]: GRTS = Generalized Random Tessellation Stratified
```
```suggestion
[^1]: GRTS-design = Generalized Random Tessellation Stratified design.
Een GRTS-raster is hiervoor de basis.
```

florisvdh commented on 02_Methodiek.Rmd:
````diff
@@ -0,0 +1,374 @@
+
+```{r workflow-rastercellen, fig.cap='(ref:workflow-rastercellen)'}
+include_graphics(path = file.path("figures", "workflow_rasterselectie.png"))
+```
````
Lijkt me OK, jij zult dit het beste kunnen beoordelen :wink:
Daarom enkel vormelijke opmerkingen:

- 'ga' is wat onduidelijk, bv. in 'ga in GRTS-rasters' lijkt het een werkwoord. Misschien tussen ' ' zetten, ...
- 'GIS' wordt het vaakst gebruikt voor dedicated software pakketten, maar sensu lato is de term zeker juist. Omdat het door die verwarring kan overkomen als 'ik heb hocus-pocus gedaan in arcgis' lijkt het me hier duidelijker om 'GIS:' te vervangen door 'geografische' of 'geospatiale'.

florisvdh replied:
Zeker ergens in de methodiek opnemen: _de verwerking is volledig gebeurd met R versie 3.6.1_ , met daarbij een literatuurverwijzing naar R. Voor dat laatste, gebruik de output van `citation()`.

Voorts kun je best iets toevoegen als: _de code voor analyse en dit rapport is te vinden op de GitHub-repository [n2khab-droogtemeetnet](https://github.com/inbo/n2khab-droogtemeetnet)_.

Beide zaken kun je wellicht vooraan in methoden vermelden, bv. apart en direct onder de titel van hfst. 2 'gebruikte methodieken'.

+ je kunt daar verder verwijzen naar bijlage `\@ref(gebruikte-werkomgeving)`.

w-jan replied:
OK in die zin je goede suggesties verwerkt

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+#Gebruikte methodieken
```
```suggestion
# Methode
```

florisvdh commented on 02_Methodiek.Rmd:
````diff
@@ -0,0 +1,374 @@
+
+```{r workflow-selectiepb-part1, fig.cap='(ref:workflow-selectiepb1)'}
+include_graphics(path = file.path("figures", "workflow_pbselectie_part1.png"))
+```
````
analoge opm als in vorige fig

florisvdh replied:
'één GT in een straal van 3 m': wellicht duidelijker indien je vooraan 'voorwaarde: slechts' toevoegt

w-jan replied:
ok, figuur werd in die zin aangepast

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+De beschikbare terreinwaarnemingen in Watina zijn meestal ontoereikend. 
+Door middel van een tijdreeksanalyse (zie \@ref(tijdreeksanalyse)) kunnen de tijdreeksen met modelgegevens aangevuld worden, waardoor het wel mogelijk wordt om de indicator te berekenen.
+
+Het zal bijgevolg op basis van de tijdreeksanalyse zijn dat meetpunten als geschikt (er is tijdsmodel mogelijk) of niet geschikt (een model is niet mogelijk) zullen bevonden worden.
```
```suggestion
Het zal bijgevolg op basis van de tijdreeksanalyse zijn dat meetpunten als geschikt (een tijdsmodel is mogelijk) of niet geschikt (een tijdsmodel is niet mogelijk) zullen bevonden worden.
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+Het zal bijgevolg op basis van de tijdreeksanalyse zijn dat meetpunten als geschikt (er is tijdsmodel mogelijk) of niet geschikt (een model is niet mogelijk) zullen bevonden worden.
+
+Het vergt veel tijd om op alle beschikbare meetpunten een tijdreeksanalyse uit te voeren.
+We willen dit beperken tot de *relatief (beschouwd op het niveau van combinatie rastercel en GT-groep)* kwalitatief beste meetreeksen. 
```
```suggestion
We beperken dit daarom tot de *relatief (beschouwd op het niveau van combinatie rastercel en GT-groep)* kwalitatief beste meetreeksen. 
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+We delen de meetpunten op in een aantal kwaliteitsklassen op basis van volgende **[basiskwaliteitscriteria]{#basiskwaliteitscriteria}** :
+
+(@) De meetpunten hebben vanaf 2000 voor minstens één jaar een goede tijdreeks[^3], die een toelaat een lg3 te berekenen.
```
```suggestion
(@) De meetpunten hebben vanaf 2000 voor minstens één jaar een goede tijdreeks[^3], die een toelaat een zogenaamde lg3-waarde[^3] te berekenen.
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+We willen dit beperken tot de *relatief (beschouwd op het niveau van combinatie rastercel en GT-groep)* kwalitatief beste meetreeksen. 
+We leggen geen absolute minimum-criteria op. Indien in een rastercel voor een bepaalde GT-groep de langste tijdreeks één jaar is, dan zal deze peilbuis toch voor een tijdreeksanalyse geselecteerd kunnen worden. De kans wordt uiteraard kleiner dat met deze beperkte data een betrouwbaar model kan gebouwd worden.
+
+We delen de meetpunten op in een aantal kwaliteitsklassen op basis van volgende **[basiskwaliteitscriteria]{#basiskwaliteitscriteria}** :
```
```suggestion
We delen de meetpunten daarom op in een aantal kwaliteitsklassen op basis van volgende **[basiskwaliteitscriteria]{#basiskwaliteitscriteria}** :
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+(@) Meetpunten met een _recente_ kwalitatief goede tijdreeks primeren boven meetpunten met een oudere tijdreeks. 
+
+De meetpunten worden hierbij gerangschikt volgens aflopend jaartal (eindjaar) van de laatst beschikbare kwalitatief goede tijdreeks. 
```
inspringen:
```suggestion
  De meetpunten worden hierbij gerangschikt volgens aflopend jaartal (eindjaar) van de laatst beschikbare kwalitatief goede tijdreeks. 
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+De meetpunten worden hierbij gerangschikt volgens aflopend jaartal (eindjaar) van de laatst beschikbare kwalitatief goede tijdreeks. 
+
+Meetpunten worden als evenwaardig beschouwd als het verschil niet groter dan **`r toelaatbare_spreiding_jaren`** jaar is.
```
inspringen:
```suggestion
  Meetpunten worden als evenwaardig beschouwd als het verschil niet groter dan **`r toelaatbare_spreiding_jaren`** jaar is.
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+(@) Meetpunten met een _langere_ tijdreeks, t.t.z. een groter aantal kwalitatief goede meetjaren, krijgen ook een hogere prioriteit.
+
+Meetpunten waarvan het aantal niet meer dan **`r toelaatbaar_verschil_lengte_tijdreeks`** jaar van elkaar verschilt, worden als evenwaardig beschouwd.
```
inspringen:
```suggestion
  Meetpunten waarvan het aantal niet meer dan **`r toelaatbaar_verschil_lengte_tijdreeks`** jaar van elkaar verschilt, worden als evenwaardig beschouwd.
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+<!-- Op basis van de kwaliteit van de tijdreeksen kunnen dan per GT-groep de rastercellen in één of meer van de volgende categorieën opgedeeld worden. Deze categorieën sluiten elkaar niet mutueel uit (zie onder). -->
+
+
+**_vraag: wat te doen met bestaande meetlocaties die actueel niet geschikt zijn? Worden deze weer geactiveerd (gevolgd in deze versie) of geven we voorrang aan de ruimtelijke balans ?_**
```
```suggestion
**_vraag: wat te doen met bestaande meetlocaties die actueel niet geschikt zijn? Worden deze weer geactiveerd (gevolgd in het huidige voorstel) of geven we voorrang aan de ruimtelijke balans ?_**
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+Het onderscheiden van kwaliteitsverschillen heeft vooral een impact op de behandeling van de twee andere categorieën. 
+
+De grootte van deze clusters wordt dan vergeleken met het gewenste aantal gewenste meetlocaties voor een rastercel. 
```
```suggestion
De grootte van deze clusters wordt dan vergeleken met het aantal gewenste meetlocaties voor een rastercel, per GT-groep. 
```

florisvdh commented on 02_Methodiek.Rmd:
````diff
@@ -0,0 +1,374 @@
+```{r voorbeeld-selectie-part1, fig.cap="Cijfervoorbeeld van de gevolgde workflow bij de selectie", , fig.width=8, fig.height=8, eval = FALSE}
+  include_graphics(path = file.path(".","figures", 
+                                    "voorbeeld_selectie_part1.png")) 
+```
````
Witregels gebruikten om zaken te scheiden! Ik vermoed dat het ontbreken van een witregel onder r. 217 (scheiden van 2 chunks) ertoe leidde dat beide figuren niet getoond worden.

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+Minstens twee derden van het peilverloop dient dus verklaard te kunnen worden door het weer.
+Eén derde kan/mag door andere factoren beïnvloed worden (bijv. het naburige peil van een waterloop).
+
+Een [lineaire trend]{#lintrend} werd als significant beschouwd indien de stijging/daling per jaar (= `trend`), rekening houdend met de standaardafwijking hierop (= `se`), voldoet één van de volgende twee regels:
```
```suggestion
Een [lineaire trend]{#lintrend} werd als significant beschouwd indien de stijging/daling per jaar (= `trend`) voldoet aan één van de volgende twee overschrijdingen in een éénzijdige statistische toets (met `se` de standaardfout van de trendschatting):
```

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+Een [lineaire trend]{#lintrend} werd als significant beschouwd indien de stijging/daling per jaar (= `trend`), rekening houdend met de standaardafwijking hierop (= `se`), voldoet één van de volgende twee regels:
+
+$$trend - 1.96* se > 1(cm/jaar)$$ 
```
evenwaardig, maar nog vlotter te interpreteren:
```suggestion
$$trend > 1 (cm/jaar) + 1.96* se$$ 
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+$$trend - 1.96* se > 1(cm/jaar)$$ 
+of 
+$$trend + 1.96* se < -1(cm/jaar)$$
```
```suggestion
$$trend < -1 (cm/jaar) - 1.96* se$$
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+$$trend - 1.96* se > 1(cm/jaar)$$ 
+of 
+$$trend + 1.96* se < -1(cm/jaar)$$
+Er is dan 2.5% kans een trendwaarde die groter resp. kleiner is ten onrechte als een significante trend te bestempelen.
```
```suggestion
Er is dan bij benadering, per statistische toets, 2.5% kans een trendwaarde die groter resp. kleiner is _ten onrechte_ als een significante trend te bestempelen (type I fout).
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+Er werden geen kwaliteitseisen gesteld aan een meetreeks die deze a priori zou uitsluiten van een tijdreeksanalyse.
+Het is, door het hoge aantal beschikbare meetpunten, echter niet doenbaar om een tijdreeksanalyse uit te voeren op alle meetpunten. 
+
+Wel werden alle meetreeksen van de meetpunten die behoren tot rastercellen van cat. 1B geanalyseerd. 
```
> cat. 1B

is niet langer gedefinieerd?

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+ 
+De gestelde kwaliteitseisen hebben alleen een weerslag op de categorie-indeling.
+Ze leiden niet tot uitsluiting.
+In principe kan daarom elke peilbuis in categorie 1B terecht komen, zelfs is er maar één meting van bekend.
```
> categorie 1B

is niet langer gedefinieerd?

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+Afhankelijk van de verhouding tussen het aantal gewenste en beschikbare peilbuizen, moeten er nog enkele bijkomende GRTS-analysen uitgevoerd worden.
+
+##### Selecteren van potentiële geschikte habitatvlekken voor rastercellen waarvoor er te weinig peilbuizen bestaan (cat. 1) {#sel-pot-habvlek}
```
```suggestion
##### Selecteren van potentieel geschikte habitatvlekken voor rastercellen waarvoor er te weinig peilbuizen bestaan (cat. 1) {#sel-pot-habvlek}
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+Na de ruimtelijke selectie kennen we hoeveel meetpunten er per rastercel en per GT-groep gewenst zijn.
+We proberen die wensen zoveel mogelijk in te vullen met bestaande peilbuizen waarvan hun meetreeksen toelaten een droogte-indicator te berekenen.
+
+Hiertoe evalueren (zonder te selecteren) we de kwaliteit van de meetreeksen op basis van de mogelijkheid om voor minstens één jaar een lg3 te berekenen voor de metingen na 2000, van het aantal meetjaren en van de ouderdom van de meetreeks. 
```
```suggestion
Hiertoe evalueren we (zonder te selecteren) de kwaliteit van de meetreeksen op basis van 1) de mogelijkheid om voor minstens één jaar een lg3^[3] te berekenen met de metingen na 2000, 2) het aantal meetjaren en 3) de ouderdom van de meetreeks. 
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+Het zijn de meetpunten die in rang volgen op de laagste rang van een gekozen punt.
+
+#### Samenvatting werkgang selectie peilbuizen
+Na de ruimtelijke selectie kennen we hoeveel meetpunten er per rastercel en per GT-groep gewenst zijn.
```
```suggestion
Na de ruimtelijke selectie weten we hoeveel meetpunten er per rastercel en per GT-groep gewenst zijn.
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+Hiertoe evalueren (zonder te selecteren) we de kwaliteit van de meetreeksen op basis van de mogelijkheid om voor minstens één jaar een lg3 te berekenen voor de metingen na 2000, van het aantal meetjaren en van de ouderdom van de meetreeks. 
+
+De kwalitatief beste meetreeksen worden verder beoordeeld met een tijdreeksanalyse. Door het grote aantal peilbuizen was het niet mogelijk ze alle te analyseren. De voorkeur ging daarom uit naar de relatief beste reeksen. Een meetreeks wordt 'geschikt' bevonden als minstens 2/3 van de variatie door het weer kan verklaard worden en er in de reeks geen lineaire trend te zien is. Op basis van de tij ...
```
(zinnen splitsen over regels)
```suggestion
De kwalitatief beste meetreeksen worden verder beoordeeld met een tijdreeksanalyse. Door het grote aantal peilbuizen was het niet mogelijk ze alle te analyseren. De tijdreeksanalyse gebeurt daarom eerst voor de beste meetreeksen. Een meetreeks wordt 'geschikt' bevonden als minstens 2/3 van de variatie door het weer kan verklaard worden en er in de reeks geen lineaire trend te zien is. Op basis van de tijdreeksanalyse worden de meetreeksen geherklasseerd in 'geschikt', 'niet geschikt' en 'onbekend' (indien niet behandeld). Er worden minstens zoveel peilbuizen geanalyseerd tot er evenveel of meer geschikte reeksen zijn dan het gewenste aantal, zoniet worden ze allemaal geanalyseerd.
```

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+Deze werkwijze laat ook gemakkelijk toe om reserve-punten te kiezen.
+Het zijn de meetpunten die in rang volgen op de laagste rang van een gekozen punt.
+
+#### Samenvatting werkgang selectie peilbuizen
```
Ik raad aan om dit stuk te vervroegen, naar net vóór de titel 'Indeling van rastercellen in categorieën', en als volgt in te leiden.

```suggestion
#### Overzicht werkgang selectie peilbuizen

In de volgende paragrafen wordt in detail uiteengezet hoe we komen tot een selectie van peilbuizen per rastercel en GT-groep (en zonodig tot bijkomende nieuwe locaties), uitgaande van de waarden `B` en `G`.
Daarom schetsen we hier eerst een overzicht van deze workflow - zie ook figuur \@ref(fig:workflow-selectiepb-part2).

```

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

florisvdh commented on lit_droogtemeetnet.bib:
```diff
@@ -0,0 +1,38 @@
+   year = {2018},
+   type = {Book}
+}
+
```
```suggestion

@misc{vanderhaeghe_grtsmh_diffres_2019,
  title = {{{GRTSmh}}\_diffres: The Raster Data Source {{GRTSmaster}}\_habitats Converted to 9 Hierarchical Cell Address Levels at the Corresponding Lower Resolution},
  shorttitle = {{{GRTSmh}}\_diffres},
  language = {eng},
  publisher = {{Zenodo}},
  author = {Vanderhaeghe, Floris},
  month = jul,
  year = {2019},
  doi = {10.5281/zenodo.3354406}
}

@misc{onkelinx_grts_2019,
  title = {{{GRTS}} Master Sample for Habitat Monitoring in {{Flanders}}},
  publisher = {{Zenodo}},
  author = {Onkelinx, Thierry and Westra, Toon and Vanderhaeghe, Floris},
  month = may,
  year = {2019},
  keywords = {GRTS,Natura 2000 habitat,Survey design},
  doi = {10.5281/zenodo.2682323}
}

```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+Hoe kleiner de celgrootte, hoe beter de ruimtelijke balans kan zijn, omdat men minder vrij is om een locatie te kiezen, maar ook hoe kleiner de kans is om daar een geschikt meetpunt aan te treffen. 
+Hier is het dus wat zoeken naar een goede maat. 
+Concreet:  
+    - We maken gebruik van een set gekoppelde GRTS-rasters op een verschillende resolutie ([GRTSmh_diffres](https://doi.org/10.5281/zenodo.3354405)) die zijn afgeleid van het basisraster 'GRTSmaster_habitats' ([link](https://doi.org/10.5281/zenodo.2682323)).
```
Heb deze opgenomen in de literatuurlijst:

```suggestion
    - We maken gebruik van een set gekoppelde GRTS-rasters op een verschillende resolutie [@vanderhaeghe_grtsmh_diffres_2019] die zijn afgeleid van het basisraster 'GRTSmaster_habitats' [@onkelinx_grts_2019].
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+In cat. 2 klopt het gewenste aantal met het beschikbaar aantal en hoeft er geen aansluitende analyse meer uitgevoerd te worden.
+In cat. 3 zijn er meer peilbuizen beschikbaar. Afhankelijk van de verhouding tussen gewenste aantal en het aantal geschikte peilbuizen is er mogelijk een extra analyse nodig. 
+
+- Is het aantal geschikte peilbuizen groter dan wordt enkel binnen deze groep een GRTS-analyse uitgevoerd om het gewenste aantal te selecteren.
```
```suggestion
- Is het aantal geschikte peilbuizen groter dan het gewenste aantal, dan wordt enkel binnen deze groep een GRTS-analyse uitgevoerd om het gewenste aantal te selecteren.
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+- Is het aantal geschikte peilbuizen groter dan wordt enkel binnen deze groep een GRTS-analyse uitgevoerd om het gewenste aantal te selecteren.
+
+- Is het aantal kleiner dan wordt er uit alle beschikbare peilbuizen via een GRTS-analyse het gewenste aantal geselecteerd.
```
```suggestion
- Is het aantal kleiner dan het gewenste aantal, dan wordt er uit alle beschikbare peilbuizen via een GRTS-analyse het gewenste aantal geselecteerd.
```

florisvdh commented on 02_Methodiek.Rmd:
```diff
@@ -0,0 +1,374 @@
+
+- Is het aantal kleiner dan wordt er uit alle beschikbare peilbuizen via een GRTS-analyse het gewenste aantal geselecteerd.
+
+- Is het aantal gelijk dan hoeft er geen verdere analyse te gebeuren. Alle meetpunten kunnen in het meetnet worden opgenomen.
```
```suggestion
- Is het aantal gelijk aan het gewenste aantal, dan hoeft er geen verdere analyse te gebeuren. Alle meetpunten kunnen in het meetnet worden opgenomen.
```

florisvdh reviewed (commented):
good work @w-jan ! :+1: 
Various suggestions etc below.

florisvdh reviewed (commented):
good work @w-jan ! :+1: 
Various suggestions etc below.

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

w-jan committed:
Update lit_droogtemeetnet.bib

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

w-jan committed:
Update 02_Methodiek.Rmd

Co-Authored-By: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

w-jan committed:
some small updates 01_Inleiding.Rmd, 02_Methodiek.Rmd, 03_Uitvoering.Rmd

tm-figures improve aspect ratio
improve workflow figures
add literature-references

w-jan committed:
02_Methodiek.Rmd some write improvements

Merge branch 'grts_selection' of https://github.com/inbo/n2khab-droogtemeetnet into grts_selection

# Conflicts:
#	02_Methodiek.Rmd

w-jan committed:
update Rmd-files for publishing a pdf

Unfortunately one get this error message:
"! Dimension too large.
\@currbox ->\bx@Q"

w-jan committed:
add workflow images in portrait

these were intended to be used in the pdf-version.

w-jan committed:
small update of the description
